mixxx-now-playing.sh

```diff
@@ -10,37 +10,73 @@ OS=`uname`
 
 echo "Detected OS: $OS"
 
-while true; do
-	while pgrep -i mixxx > /dev/null; do
-	
+quartzListAllWindows=$'import Quartz\nprint(Quartz.CGWindowListCopyWindowInfo(Quartz.kCGWindowListExcludeDesktopElements|Quartz.kCGWindowListOptionAll,Quartz.kCGNullWindowID))'
+
+lastKnownTitle=" "
+
+FormatAndWriteTitle(){
+	currentTitle=$(  
+	echo "$1" |
+	cut -d\| -f1 |
+	sed 's/,/ -/' |
+	awk '{ print tolower($0) }' |
+	ascii2uni -aU -q|
```

Comment:
doesn't work with some characters such as in
V ∴ V ∴ V (Headless Horseman Remix), 
¿Can You Hear Me?

when a song title with these characters show up sed barfs with "sed: RE error: illegal byte sequence"

Reply:
That's interesting, the sed line has not been altered from the original.

Reply:
I just pushed a change to move the unicode conversion much earlier in the flow, which will hopefully fix the issues you're seeing. Unfortunately, I don't have an environment where I can test it, so it's a blind change.